src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveLevelResource.kt

```diff
@@ -266,16 +270,16 @@ class IncentiveLevelResource(
 
   @DeleteMapping("levels/{code}")
   @Operation(
-    summary = "Disables an incentive level",
-    description = "",
+    summary = "Deactivates an incentive level",
+    // TODO: decide and explain what happens to associated per-prison data
     responses = [
       ApiResponse(
         responseCode = "200",
-        description = "Incentive level disabled"
+        description = "Incentive level deactivated"
       ),
       // ApiResponse(
       //   responseCode = "400",
```

Comment:
Is this meant to be commented?

Reply:
yes, i've deliberately left lots of TODOs as discussion points! once all the business rules have been determined, then can get rid of all of these